src/hub/ShareClassManager.sol

```diff
@@ -768,7 +793,7 @@ contract ShareClassManager is Auth, IShareClassManager {
             poolId,
             scId_,
             assetId,
-            issueEpochId[scId_][assetId] + 1,
+            epochId[scId_][assetId].deposit + 1,
```

Comment:
@wischli @mustermeiszer I believe this was a mistake and should be deposit rather than issue ID?

Reply:
Yes, good catch! There are also the `nowDepositEpoch()` getters.